### DISCUSSION TITLE
Some additional details about the database dump

### DISCUSSION THREAD

**Discussion by szabgab** [rust-lang/crates.io#7030] [Q&A] [answered]
I have  been using the experimental database dump mentioned here https://crates.io/data-access to build the [Rust Digger](https://rust-digger.code-maven.com/). I think I understand most of the schema, but there are a number of places where some clarification would help.

* In the `teams.csv` file there is a column called `login` with values such as `github:mmtk:release-team` What does this mean?
* In the `crate_owners.csv` file there is a column called `owner_kind` with values `0` or `1`. What do they mean?
* In the `crate_owners.csv` file there is a column called `owner_id`. If I am not mistaken this either refers to the `id` field in `users.csv` or the `id` field in `teams.csv`. Is that right? If so, then I think I encountered 11 crates that don't map to any owner.

The explanations could be probably added to https://github.com/rust-lang/crates.io/blob/main/src/worker/dump_db/readme_for_tarball.md that comes with the dump.

If I am not mistaken #2078 discusses the same exported data.

**Comment by Turbo87** [comment, marked as answer]
> * In the `crate_owners.csv` file there is a column called `owner_kind` with values `0` or `1`. What do they mean?

- `owner_kind: 0` means that this is referring to a `user`
- `owner_kind: 1` means that this is referring to a `team`

> * In the `crate_owners.csv` file there is a column called `owner_id`. If I am not mistaken this either refers to the `id` field in `users.csv` or the `id` field in `teams.csv`. Is that right?

that is correct


> * In the `teams.csv` file there is a column called `login` with values such as `github:mmtk:release-team` What does this mean?

it is referring to the `release-team` team of the `mmtk` GitHub organization.
